app/javascript/vue/tasks/leads/hub/app.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <h1>Dichotomous Keys Hub</h1>
+  <h1>Multifurcating Keys Hub</h1>
```

Comment:
I think we're going to use "Keys hub" and "Interactive keys hub" as the differentiation.